ui/src/buckets/components/BucketList.tsx

```diff
@@ -91,7 +95,13 @@ class BucketList extends PureComponent<Props & WithRouterProps> {
 
     return sortedBuckets.map(bucket => {
       if (bucket.type === 'demodata') {
-        return <DemoDataBucketCard key={bucket.id} bucket={bucket} />
+        return (
+          <DemoDataBucketCard
+            key={bucket.id}
+            bucket={bucket}
+            onRemoveBucket={deleteDemoDataBucketMembership}
```

Comment:
this should go into the demodatabucketcard component and not be added through a prop. also, don't you need a `connect` for actions to work?

Reply:
it's prop drilled further than you think!!

Reply:
but I'm fixing it.